interpreter/cling/lib/Utils/AST.cpp

```diff
@@ -1693,6 +1693,7 @@ namespace utils {
     PrintingPolicy Policy(Ctx.getPrintingPolicy());
     Policy.SuppressScope = false;
     Policy.AnonymousTagLocations = false;
+    Policy.FullyQualifiedName = true;
```

Comment:
Please make sure that this does not hide a missing feature in GetFullyQualifiedType which is suppose to already have built-in the full qualification.